bin/yb-ctl

```
@@ -1022,48 +1070,135 @@ class ClusterControl:
+                num_alive_ts, num_yb_admin_ts))
+            return False
+        return True
 
     def wait_for_cluster_or_raise(self):
         if not self.wait_for_cluster():
             raise RuntimeError("Timed out waiting for Yugabyte cluster!")
 
-    def show_node_status(self, daemon_id):
-        pid = self.get_pid(daemon_id)
-
-        def get_address(port_type):
-            return self.options.get_address(daemon_id, port_type, include_base_url=True)
-
-        if pid is None:
-            logging.info("Server {} is not running".format(daemon_id))
-        else:
-            info_list = [
-                ("type", daemon_id.daemon_type),
-                ("node_id", daemon_id.index),
-                ("PID", pid),
-                ("admin service", "http://" + get_address("http"))
-            ]
-
-            if daemon_id.is_tserver():
-                info_list.extend([
-                    ("cql service", get_address("cql_rpc")),
-                    ("redis service", get_address("redis_rpc")),
-                ])
-                if self.is_postgres_enabled():
-                    info_list.append(("pgsql service", get_address("pgsql_rpc")))
-            logging.info("Server is running: {}".format(
-                ", ".join(["{}={}".format(k, v) for k, v in info_list])))
+    def query_cluster_uuid(self):
+        cluster_uuid = None
+        try:
+            with open(os.devnull, "wb") as devnull:
+                cluster_uuid = subprocess.check_output(
+                    "curl localhost:7000/cluster-config | grep cluster_uuid",
+                    shell=True,
+                    stderr=devnull).strip()
+            regex_groups = re.match('cluster_uuid: "(.*)"', cluster_uuid).groups()
+            # We will display None if the subprocess fails...
+            cluster_uuid = regex_groups[0] if regex_groups else None
+        except subprocess.CalledProcessError as e:
+            pass
```

Ditto regarding suppressing errors.